src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/TaskUtilities.cs

```diff
@@ -41,5 +42,16 @@ public static Task<int> GetCancelledZeroTask(CancellationToken cancellationToken
             return tcs.Task;
 #endif
         }
+
+        public static Task<int> GetFaultedTask(Exception error, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Thanks, I had copied the method above it and forgot to remove the unused arg :smile: